docs/design/network-discovery/disable-port-discovery/disable-port-discovery.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+title Port discovery enable/disable
+
+participant Northbound as NB
+participant nbworker as NBW
+database DB
+control kilda.topo.disco as DI
+box NetworkTopology
+participant SpeakerRouter as SR
```

Comment:
As I understood, SpeakerRouter is for requests that come from Speaker, not NB.

Reply:
For now SpeakerRouter is used for at least two NB messages: IslBfdFlagUpdated and FeatureTogglesUpdate. Probably we need to rework topology or rename SpeakerRouter.

Reply:
there is a comment on that      // FIXME(surabujin): is it ok to consume this "event" from speaker stream?

Reply:
P.S. I' refactor that separately